shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/ddl/dataset/encrypt/change_user_table.xml

```diff
@@ -0,0 +1,24 @@
+<!--
```

Comment:
I assume `alter_user_user` is better to name this file.

Reply:
how about change_column_user_table.xml ?
Because there are other alter operations, I'm afraid it will cause confusion.

Reply:
> I assume `alter_user_user` is better to name this file.

how about change_column_user_table.xml ?
Because there are other alter operations, I'm afraid it will cause confusion.